src/Components/CreateImageWizardV2/steps/Oscap/Oscap.tsx

```diff
@@ -0,0 +1,274 @@
+import React, { useEffect, useState } from 'react';
+
+import {
+  Alert,
+  FormGroup,
+  Spinner,
+  Popover,
+  TextContent,
+  Text,
+  Button,
+} from '@patternfly/react-core';
+import {
+  Select,
+  SelectOption,
+  SelectVariant,
+} from '@patternfly/react-core/deprecated';
+import { HelpIcon } from '@patternfly/react-icons';
+
+import OscapProfileInformation from './OscapProfileInformation';
+
+import { useAppDispatch, useAppSelector } from '../../../../store/hooks';
+import {
+  DistributionProfileItem,
+  useGetOscapCustomizationsQuery,
+  useGetOscapProfilesQuery,
+} from '../../../../store/imageBuilderApi';
+import {
+  changeOscapProfile,
+  selectDistribution,
+  selectProfile,
+} from '../../../../store/wizardSlice';
+
+/**
+ * Component for the user to select the profile to apply to their image.
+ * The selected profile will be stored in the `oscap-profile` form state variable.
+ * The Component is shown or not depending on the ShowSelector variable.
+ */
```

Comment:
This was true for the DDF version, right? We should either update the comment or remove it.